portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,25 +23,42 @@
 import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+// @WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String saying = "Hello Again";
-    ArrayList<String> messages = new ArrayList<String>();
-    messages.add("message one");
-    messages.add("message two");
-    messages.add("message three");
-    String json = convertToJsonUsingGson(messages);
-
-    response.setContentType("text/html;");
-    response.getWriter().println(json);
+//   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Are you sure you need to comment this all out? I think it should be okay to define handlers for both Get and Post.

Or maybe /data and /comments should have separate Servlets?

Reply:
I think I will keep them on the same servlets, I just commented them out here because I was just focused on the Post handler for this pull request and I wanted to make sure I knew how it worked

Reply:
Sounds like the weird state of things in this PR is because of the tutorial you're following. So that's fine.